libs/tree-sitter-wing/.gitignore

```diff
@@ -1,4 +1,5 @@
 /src/*
 !/src/scanner.c
+!/src/grammar.json
```

Comment:
will this get automatically updated by the self mutation in case it's changed?

Reply:
Yep it should